x-pack/plugins/security_solution/public/common/components/item_details_card/index.tsx

```diff
@@ -52,23 +52,26 @@ const DetailsSection = styled(EuiFlexItem)`
 `;
 
 const DescriptionListTitle = styled(EuiDescriptionListTitle)`
-  width: 40%;
+  width: 40% !important;
```

Comment:
any other way around `!important` ?  Can you put a specific class on the component to make it specific enough?

Reply:
Fixed.